.github/workflows/jmh-benchmark.yml

```diff
@@ -0,0 +1,101 @@
+name: JMH Benchmark
+
+on:
+  issue_comment:
+    types: [created]
+
+
+jobs:
+  setup:
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '/benchmark')
+    runs-on: ubuntu-latest
+    permissions: write-all
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Get PR branch
+      uses: xt0rted/pull-request-comment-branch@v1
+      id: comment-branch
+
+
+    - name: Set branch name
+      run: echo "BRANCH_NAME=${{ steps.comment-branch.outputs.head_ref }}" >> $GITHUB_ENV
+    
+    - id: 'auth'
+      name: Authenticating
+      uses: 'google-github-actions/auth@v1'
+      with:
+        credentials_json: '${{ secrets.GCP_SA_KEY }}'
+
+    - name: Set up Google Cloud SDK
+      uses: google-github-actions/setup-gcloud@v1
+      
+      
+    - name: Setup Directory and Copy Bash Scripts
+      run: |
+        gcloud compute ssh \
+          --project=nullway-jmh \
+          --zone=us-central1-a \
+          root@instance-1 \
+          --command="export BRANCH_NAME=${BRANCH_NAME} && mkdir $BRANCH_NAME"
+        gcloud compute scp ./.github/workflows/run_pr_benchmarks.sh root@instance-1:$BRANCH_NAME/ --zone=us-central1-a
+        gcloud compute scp ./.github/workflows/run_main_benchmarks.sh root@instance-1:$BRANCH_NAME/ --zone=us-central1-a
+
+    - name: Run JMH benchmarks for PR Branch
+      run: |
+        gcloud compute ssh \
+          --project=nullway-jmh \
+          --zone=us-central1-a \
+          root@instance-1 \
+          --command=" export BRANCH_NAME=${BRANCH_NAME} && chmod +x $BRANCH_NAME/run_pr_benchmarks.sh && $BRANCH_NAME/run_pr_benchmarks.sh"
+          
+    - name: Copy exported PR benchmark file from Google Cloud Compute Engine
+      run: |
+        gcloud compute scp root@instance-1:$BRANCH_NAME/pr/NullAway/jmh/build/results/jmh/results.txt ./pr_text.txt --zone us-central1-a
+        
+    - name: Formatting Benchmark
+      run:  |
+        (echo 'PR Benchmark:'; echo '```' ; cat pr_text.txt; echo '```') > pr.txt
+   
+    - name: Comment Benchmark
```

Comment:
I'd prefer to run benchmarks on both the PR branch and the main branch, and then add a single PR comment with the results from both

Reply:
Fixed